### PR TITLE
Add symptom selection functionality to symptom checker

### PR DESCRIPTION
**Pull request by devinjameson**
Why: as a user, I would like to be able to select some symptoms and be given a recommendation on how to proceed.

This commit:
- Adds the ability for a user to select symptoms from a list
- Gives the user a recommendation based on their symptoms. Right now, the scoring is just based on whether the user has selected more than 3 symptoms. Of course, this is a placeholder.
- Adds a screen that the user is navigated to if their symptoms indicate they may be at risk
- Adds a `SymptomCheckerContext`

Co-Authored-By: Alejandro Dustet <alejandro@thoughtbot.com>
Co-Authored-By: John Schoeman <johnschoeman1617@gmail.com>

gif2

### CODE REVIEW

File: src/SymptomChecker/CheckIn.tsx [hunk 1]
```diff
@@ -121,13 +138,44 @@ const FeelingGoodContent = () => {
...
+
+const FeelingNotWellContent: FunctionComponent<FeelingNotWellContentProps> = ({
+  symptoms,
+  healthAssessment,
+}) => {
   const { t } = useTranslation()
 
+  const determineHealthAssessmentText = () => {
+    switch (healthAssessment) {
+      case HealthAssessment.GetTested:
```
Comment (johnschoeman) [inline src/SymptomChecker/CheckIn.tsx]:
```suggestion
      case HealthAssessment.AtRisk:
```

File: src/SymptomChecker/CheckIn.tsx [hunk 2]
```diff
@@ -121,13 +138,44 @@ const FeelingGoodContent = () => {
...
+        return t("symptom_checker.get_tested")
+      case HealthAssessment.FollowHAGuidance:
```
Comment (johnschoeman) [inline src/SymptomChecker/CheckIn.tsx]:
```suggestion
      case HealthAssessment.NotAtRisk:
```

This naming will leave us open to keep the notion of the assessment flow separate from the recommendation flow and allow them to grow/change independently. for example we might have a future world where we want something like

```typescript
type HealthAssessment = NotStarted | InProgress | AtRisk | NotAtRisk

type HealthRecommendation = GetTested | CallDoctor | DoNothing
```

its a bit premature to finalize what exactly these states should be but we probably want to be moving towards something like this to support the self assessment feature.

**Review by johnschoeman — APPROVED** [review]
LGTM!

File: src/SymptomChecker/CheckIn.tsx (hunk 2 above)
Comment (devinjameson) [reply src/SymptomChecker/CheckIn.tsx]: Sounds good!

**Pull request merged**